text/0000-security-advisories.md

```diff
@@ -0,0 +1,259 @@
+- Feature Name: security_advisories
+- Start Date: 2016-08-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io should offer an API to release security advisories for crates. Cargo
+should offer a new command for the same purpose and warn about vulnerable crate
+versions during compilation.
+
+# Motivation
+[motivation]: #motivation
+
+When compared to other ecosystems such as Python's, Rust's broader community
+prefers many single-purpose crates over larger monoliths. This situation,
+together with the strongly encouraged practice of pinning MINOR versions of
+dependencies, slows down the propagation of critical security fixes.
+
+Assume a crate `W`, which depends on `X`, which depends on `Y`, which depends
+on `Z`.  If `Z` releases a new MINOR version including a security fix, it
+requires the attention of `Y`'s  and `X`'s maintainers to propagate that
+security fix to `W`. What makes this situation worse is that the author of `W`
```

Comment:
I guess if the fix requires a major version bump, you get the same scenario as with pinned versions.